rstream/client.py

```diff
@@ -172,9 +180,22 @@ async def start(self) -> None:
         self._conn = Connection(self.host, self.port, self._ssl_context)
         await self._conn.open()
         self.start_task("listener", self._listener())
+        self.start_task("run_delivery_handlers", self._run_delivery_handlers())
         self.add_handler(schema.Heartbeat, self._on_heartbeat)
         self.add_handler(schema.Close, self._on_close)
 
+    async def _run_delivery_handlers(self):
+
+        while True:
+            try:
+                frame_entry = await self._frames.get()
+                maybe_coro = await frame_entry.handler(frame_entry.frame)
```

Comment:
There's an extra `await` here

This thing with `if maybe_coro is not None` was introduced to allow to have both sync and async handlers. The equivalent code would be
```python
if asyncio.iscoroutinefunction(handler):
    await handler(frame)
else:
    handler(frame)
```
(but if the handler is sync, there's no point in moving it to another asyncio.Task)